docs/config/_default/params.yaml

```diff
@@ -5,6 +5,8 @@ github_subdir: docs
 
 versions:
   - url: /docs/
-    version: development
-  - url: https://lifecycle.keptn.sh
-    version: production
+    version: branch-preview
+  - url: https://main.lifecycle-test.keptn.sh/docs
+    version: main
+  - url: https://lifecycle-test.keptn.sh/docs
+    version: latest
```

Comment:
i am really unsure about the naming here - what is best suited

eg. for the main branch - do we want to use:
- development
- main
- preview

for the live documentation do we want to use:
- live
- latest
- current

do we want to have this at all?

Reply:
As a prefix for the `main` branch documentation, I would use `development`. 
For the live documentation, I would not use any prefix

Reply:
I don't have strong feelings about the specific terms but I do like to know which version of docs I am viewing.  Using "latest" or "current" or some other term in the URL is good because the user's bookmarks and xrefs from external sources persist across releases but could we maybe have some sort of text on each page that identifies the release?